scripts/build_kvm_image.sh

```diff
@@ -51,6 +51,7 @@ create_disk
 prepare_installer_disk
 
 echo "Prepare memory for KVM build: $vs_build_prepare_mem"
+sudo mount proc /proc -t proc || true
```

Comment:
Is it related to buster? If not, separate it into another PR?

Reply:
anyway, I plan not to squash merge on this pr. I plan to do a merge on this. is already a separate commit. 3df2831a9ff6572f9843c8aa793eebe7c356be1d